modules/html/src/main.rs

```diff
@@ -1,10 +1,20 @@
-use serde_json::{from_str, json, Value};
 use std::{
     env,
     fmt::Write,
     io::{self, Read},
 };
 
+use serde_json::{from_str, json, Value};
+
+macro_rules! raw {
```

Comment:
Nice : )

Reply:
Haha thanks! I have changed some other parts of the code using `raw` nodes as well, in addition to my own code